spec/spec_helper.rb

```diff
@@ -16,7 +16,7 @@
 RAW_CORRECT = YAML.safe_load(File.read('spec/fixtures/raw_scopus.yml'), [Symbol])
```

Comment:
Suggest you provide this line `ENV['RACK_ENV'] = 'test'` in this file.
Therefore, when you run `rake spec`, will not necessary add `RACK_ENV=test`